manuscript/03-Why-Service-Meshes.md

```diff
@@ -246,6 +246,12 @@ memory and CPU. However, hardware is becoming cheaper constantly and
 adding some hardware resources to improve the reliability of a system
 might be acceptable.
 
+The amount of resource consumption and latency highly depends on the 
+service mesh implementation and version, settings, and the amount of 
+network traffic of the specific application. Therefore, if 
+either is restricted by requirements, it should be measured individually in a 
+test deployment and compared to a mesh-free deployment.
```

Comment:
IMHO you should compare it to an alternative solution (i.e. a library), "mesh-free" does not express that.

Reply:
What I meant was to compare it to the application as-is since no one would (or should) invest time in implementing the features using libraries when they evaluate a service mesh. So maybe a better way to put it would be:

> and compared to the pre-mesh deployment. When comparing the numbers, keep in mind that the pre-mesh deployment lacks features. Once implemented within the applications, e.g. using libraries, resource consumption and latency would also be affected.